### DISCUSSION TITLE
Facing issue with .contains()

### DISCUSSION THREAD

**Discussion by sreenarayan** [cypress-io/cypress#27144] [Questions and Help]
Not fetching the desired element.
cy.get(".sb-dropdown--header-title",{ timeout: 10000 }).eq(2).click(); //county
cy .contains("Contra Costa", { timeout: 10000 }).click({ force: true });

Even dropdown is open, element is visible. cypress is unable to find it

Using cypress version --12.11.0

**Comment by michallbanas** [comment]
Hey hey @sreenarayan 👋

- Can you post the screenshot of failed test? 
- Did you try to debug the page and check if the element doesn't contain eg. hidden characters or needs to be trimmed etc.?
